### ISSUE TITLE
[DEFECT] On the 3-column template, the tablet breakpoint causes the site to bug out a little.

### ISSUE THREAD

**Issue by kaylaziegler** [chapmanu/cascade-assets#212]
https://trello.com/c/Yd5FflBm/443-defect-on-the-3-column-template-the-tablet-breakpoint-causes-the-site-to-bug-out-a-little-needs-fixin

For pages using the 3 column template that have right column callouts, shrinking the window to tablet and phone screen sizes cause the topmost callout to cover the page and it doesn't reposition correctly.
![screen_shot_2017-06-23_at_4 39 22_pm](https://user-images.githubusercontent.com/11510966/27970454-b8fc97e0-6303-11e7-88e6-133cb76863c7.png)

**Comment by kaylaziegler** [comment]
Ross originally reported this as a law school-specific issue but after investigating, it's actually an issue for all pages using the 3 column template. 

It appears to reposition correctly when the topmost widget in the right column is anything but styleOne, so it works fine when the callout on top uses styleTwo or if there are buttons on top of the right column.
Correct examples:
- https://www.chapman.edu/students/health-and-safety/health-services/index.aspx
- https://www.chapman.edu/law/admission/apply-now.aspx

Incorrect examples:
- https://www.chapman.edu/law/student-resources/computing-email/index.aspx
- https://www.chapman.edu/students/health-and-safety/public-safety.aspx

**Comment by kaylaziegler** [comment]
Pretty sure I found the problem. For styleOne, simple-form-widget and the logo image rotator, we did not explicity specify the `overflow` property. For styleTwo and other correctly working callouts, they have the `overflow: hidden` property. If `overflow` is not specified, the overflow will render outside the element by default. See this [informative guide](https://www.w3schools.com/cssref/pr_pos_overflow.asp) to the `overflow` property.
